eglot-jl.el

```diff
@@ -83,9 +68,8 @@ Otherwise returns nil"
   "Return list of strings to be called to start the Julia language server."
   `(,eglot-jl-julia-command
     ,@eglot-jl-julia-flags
-    ,(concat "--project=" eglot-jl-base)
+    "--project"
```

Comment:
Does eglot guarantee that the ls-invocation will be run in this directory somehow? If so, is it a guarantee of the api or an implementation detail?